### PR TITLE
Update contributing.md

### PR DESCRIPTION
**Pull request by acidus101**

### CODE REVIEW

File: contributing.md [hunk 1]
```diff
@@ -5,7 +5,7 @@ By participating in this project you agree to abide by its terms.
 
 ## Add quality
 
-Ensure your pull request adheres to the following guidelines
+Ensure your pull request adheres to the following guidelines.
```
Comment (koppor) [inline contributing.md]: Following the sentence, there is an enumeration. Please use a `:`.

**Comment by koppor** [comment]
Thank you for taking part in #hacktoberfest. Please take it as opportunity to level-up your coding skills.

File: contributing.md (hunk 1 above)
Comment (acidus101) [reply contributing.md]: sorry i clicked resolve by mistake im changing it in sometime. thank you for your response.

File: contributing.md [hunk 2]
```diff
@@ -5,7 +5,7 @@ By participating in this project you agree to abide by its terms.
 
 ## Add quality
 
-Ensure your pull request adheres to the following guidelines
+Ensure your pull request adheres to the following guidelines :
```
Comment (koppor) [inline contributing.md]: No space before the colon please.

I wonder why it seems no caution was taken when creating the PR and the update.

**Pull request closed without merging**